### ISSUE TITLE
Make travis-build respect .nvmrc for node builds

### ISSUE THREAD

**Issue by svenfuchs** [travis-ci/travis-ci#3644]
Here's a quote from an email to support:

> Is there a way to take advantage of an '.nvmrc' file in your project with Travis? It contains the version of node we want to use, so we'd rather not update both this file and our .travis.yml file every time we change it. For what it's worth, nvm knows to look at this file when you run commands without an argument, such as `nvm install`.

I think that makes sense. As far as I understand we'd want to change https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/script/node_js.rb#L16 to something like

```
          sh.if '-f .nvmrc' do
            sh.cmd 'nvm install'
          end
          sh.else do
            sh.cmd "nvm install #{version}"
          end
```

**Comment by joshk** [comment]
Love this!

@BanzaiMan could you add this to Blue?

**Comment by spencerhakim** [comment]
I think this issue should've been closed by travis-ci/travis-build#469

**Comment by BanzaiMan** [comment]
@spencerhakim Thanks.
